pkgconfig.go

```diff
@@ -17,11 +17,16 @@ func lookupGithubIfEnv(pkg string) (*PC, error) {
 	return nil, errSkipGithub
 }
 
-var lookups = map[string]func(string) (*PC, error){
-	"$GOPATH":          LookupGopath,
-	"github.com":       lookupGithubIfEnv,
-	"$PKG_CONFIG_PATH": LookupPC,
-	"<autogenerated>":  GenerateGopath,
+type pathLookupPair struct {
+	name string
+	fn   func(string) (*PC, error)
+}
+
+var lookups = []pathLookupPair{
+	{"$GOPATH", LookupGopath},
```

Comment:
@rjeczalik Want to confirm this is the correct order to check for files. `GenerateGopath` has to go at the end though.